src/clients/api/queries/getVTokenBalancesAll.ts

```diff
@@ -0,0 +1,46 @@
+import { Contract } from 'web3-eth-contract';
+
+export interface IGetVTokenBalancesAllInput {
+  venusLensContract: Contract; // @TODO: use contract type (through Typechain?)
+  account: string | undefined | null;
+  vtAddresses: string[];
+}
+
+interface IGetVTokenBalancesAllResponse extends Array<string> {
+  balanceOf: string;
+  balanceOfUnderlying: string;
+  borrowBalanceCurrent: string;
+  tokenAllowance: string;
+  tokenBalance: string;
+  vToken: string;
+}
+
+export interface IGetVTokenBalancesAllOutput {
+  balanceOf: string;
+  balanceOfUnderlying: string;
+  borrowBalanceCurrent: string;
+  tokenAllowance: string;
+  tokenBalance: string;
+  vToken: string;
+}
+
+const getVTokenBalancesAll = async ({
+  venusLensContract,
+  vtAddresses,
+  account,
+}: IGetVTokenBalancesAllInput) => {
```

Comment:
```suggestion
}: IGetVTokenBalancesAllInput): Promise<IGetVTokenBalancesAllOutput> => {
```

Reply:
This is not returning a promise. so this type is not correct.

Reply:
What happened here was odd. The contract returns an array with named properties. Rather than index the values we use the named properties somehow in the `indexBy` function these named properties on the array were missing. which would throw an error. This was a quick solution.

Reply:
I was able to remove this